website/pages/404.html

```
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="../scripts/components/global.css">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ link ] must be self closed.

(tag-self-close)